brand-guidelines/color-principles.md

```diff
@@ -24,945 +24,109 @@ A minimal palette, clear hierarchy, good information design, and ample whitespac
-                        <th>HEX</th>
-                        <td>#337E7D</td> <!-- Update the hex code -->
-                    </tr>
-                    <tr>
-                        <th>RGB</th>
-                        <td>51, 126, 125</td> <!-- Update the RGB value -->
-                    </tr>
-                    <tr>
-                        <th>PMS</th>
-                        <td>7721 @ 80%</td> <!-- Update the PMS name -->
-                    </tr>
-                    <tr>
-                        <th>CMYK</th>
-                        <td>80, 34, 49, 10</td> <!-- Update the CMYK value -->
-                    </tr>
-                </tbody>
-            </table>
-        </figcaption>
-    </figure>
-    <figure class="swatch swatch__secondary">
-        <div class="swatch_field swatch_field__teal-50"></div> <!-- Update the color in the class name -->
-        <figcaption class="swatch_label">
-            <h4 class="swatch_head">Teal 50%</h4> <!-- Update the full color name -->
-            <table class="swatch_table">
-                <tbody>
-                    <tr>
-                        <th>HEX</th>
-                        <td>#7FAEAE</td> <!-- Update the hex code -->
-                    </tr>
-                    <tr>
-                        <th>RGB</th>
-                        <td>127, 174, 174</td> <!-- Update the RGB value -->
-                    </tr>
-                    <tr>
-                        <th>PMS</th>
-                        <td>7721 @ 50%</td> <!-- Update the PMS name -->
-                    </tr>
-                    <tr>
-                        <th>CMYK</th>
-                        <td>51, 18, 30, 0</td> <!-- Update the CMYK value -->
-                    </tr>
-                </tbody>
-            </table>
-        </figcaption>
-    </figure>
-    <figure class="swatch swatch__secondary">
-        <div class="swatch_field swatch_field__teal-20"></div> <!-- Update the color in the class name -->
-        <figcaption class="swatch_label">
-            <h4 class="swatch_head">Teal 20%</h4> <!-- Update the full color name -->
-            <table class="swatch_table">
-                <tbody>
-                    <tr>
-                        <th>HEX</th>
-                        <td>#CCDFDF</td> <!-- Update the hex code -->
-                    </tr>
-                    <tr>
-                        <th>RGB</th>
-                        <td>204, 223, 223</td> <!-- Update the RGB value -->
-                    </tr>
-                    <tr>
-                        <th>PMS</th>
-                        <td>7721 @ 20%</td> <!-- Update the PMS name -->
-                    </tr>
-                    <tr>
-                        <th>CMYK</th>
-                        <td>19, 4, 11, 0</td> <!-- Update the CMYK value -->
-                    </tr>
-                </tbody>
-            </table>
-        </figcaption>
-    </figure>
-</div>
+    {% assign family = 'green' %}
```

Comment:
This could be `{% assign family = site.data.cfpb-brand-colors[1].family %}` to avoid hardcoding this value so it doesn't break if we reorder the CSV.